src/DesignKit.tsx

```diff
@@ -2632,106 +2634,291 @@ const TooltipsSection: React.FC = () => {
+          <Column align="right">Right-aligned content</Column>
+          <Column align="right">Right-aligned content</Column>
+        </Row>
+        <hr />
+        <p>
+          A <code>{'<Column>'}</code> can have a vertical <code>{'gap'}</code>
         </p>
-        <Columns>
-          <Column align="right">
-            <Button>Content</Button>
+        <p>
+          With <code>{'gap'}</code> set to 0 (default):
+        </p>
+        <Column>
+          <Row>
+            <Card>Row content</Card>
+          </Row>
+          <Row>
+            <Card>Row content</Card>
+          </Row>
+        </Column>
+        <p>
+          With <code>{'gap'}</code> set to 8:
+        </p>
+        <Column gap={8}>
+          <Row>
+            <Card>Row content</Card>
+          </Row>
+          <Row>
+            <Card>Row content</Card>
+          </Row>
+        </Column>
+        <p>
+          With <code>{'gap'}</code> set to 16:
+        </p>
+        <Column gap={16}>
+          <Row>
+            <Card>Row content</Card>
+          </Row>
+          <Row>
+            <Card>Row content</Card>
+          </Row>
+        </Column>
+        <hr />
+        <p>
+          A <code>{'<Column>'}</code> can have a <code>{'width'}</code>, which is only applied when
+          wrapped in a <code>{'<Row>'}</code>.
+        </p>
+        <strong>Row with 3 Fill Width (default) columns</strong>
+        <Row>
+          <Column width="fill">
+            <div>Fill Width</div>
+          </Column>
+          <Column width="fill">
+            <div>Fill Width</div>
+          </Column>
+          <Column width="fill">
+            <div>Fill Width</div>
+          </Column>
+        </Row>
+        <strong>Row with 3 Hug Width columns</strong>
+        <Row>
+          <Column width="hug">
+            <div>Hug Width</div>
+          </Column>
+          <Column width="hug">
+            <div>Hug Width</div>
+          </Column>
+          <Column width="hug">
+            <div>Hug Width</div>
           </Column>
-          <Column align="right">
-            <Button>Content</Button>
+        </Row>
+        <strong>Row with 3 Fixed pixel width columns</strong>
+        <Row>
+          <Column width={100}>
+            <div>Fixed Pixel Width</div>
           </Column>
-          <Column align="right">
-            <Button>Content</Button>
+          <Column width={100}>
+            <div>Fixed Pixel Width</div>
           </Column>
-        </Columns>
+          <Column width={100}>
+            <div>Fixed Pixel Width</div>
+          </Column>
+        </Row>
+        <strong>Row with 3 columns of each width type</strong>
+        <Row>
+          <Column width="fill">
+            <div>Fill Width</div>
+          </Column>
+          <Column width="hug">
+            <div>Hug Width</div>
+          </Column>
+          <Column width={100}>
+            <div>Fixed Pixel Width</div>
+          </Column>
+        </Row>
```

Comment:
nit: same as above, the demonstration would be more effective with a border/background color